docs/en/quick-answers/troubleshooting/cant-use-file-linking-toolkit-app-chrome.md

```diff
@@ -44,21 +44,21 @@ On the other hand, if you are greeted by one of these messages, it means there w
 
 ## How to quickly fix certificate issues
 
-The easiest way to circumvent those issues is to click **ADVANCED** and **Proceed to localhost (unsafe)**. This will let Chrome know that you are accepting the certificate nonetheless and will allow the Shotgun website to communicate with Shotgun Desktop. 
+The easiest way to circumvent those issues is to click **ADVANCED** and **Proceed to localhost (unsafe)**. This will let Chrome know that you are accepting the certificate nonetheless and will allow the {% include product %} website to communicate with {% include product %} Desktop. 
 
-{% include info title="Note" content="This only enables the connection between your web browser and Shotgun Desktop. The trust setting lets traffic flow between these two applications, **it does not imply trust of any other server on the internet**. Unfortunately, this process needs to be repeated on every single computer with the problem. If this doesn't solve the issue or you feel it would be too complicated to deploy to all your users, we recommend you take a look at the following steps." %}
+{% include info title="Note" content="This only enables the connection between your web browser and {% include product %} Desktop. The trust setting lets traffic flow between these two applications, **it does not imply trust of any other server on the internet**. Unfortunately, this process needs to be repeated on every single computer with the problem. If this doesn't solve the issue or you feel it would be too complicated to deploy to all your users, we recommend you take a look at the following steps." %}
```

Comment:
Hey @zamu5 -- it looks like we won't be able to use an include within the content string of another include. That's unfortunate, but not the end of the world. Mind backing out of this and other instances of the same situation elsewhere? We will just have to leave them as "Shotgun" and not use the include.